homeassistant/components/xiaomi_aqara/__init__.py

```diff
@@ -61,16 +61,18 @@
 })
 
 
-GATEWAY_CONFIG_MAC_OPT = vol.Schema({
+GATEWAY_CONFIG_MAC_OPTIONAL = vol.Schema({
```

Comment:
Why don't you define a base schema GATEWAY_CONFIG and then use `GATEWAY_CONFIG.extend({ vol.Optional(CONF_MAC): vol.Any(…)})`